contents/handbook/engineering/support-hero.md

```diff
@@ -53,52 +77,20 @@ As a business we need to ensure we are focusing support on our paying customers,
 1. [Community questions](https://posthog.com/questions/)
 1. Open Zendesk tickets for your team that have `low` priority (non-paying users)
```

Comment:
One thing to highlight from the Support RFC is that all tickets will come in as Normal or High priority with the aim that whoever is in this role will at least look at every ticket to see if there are any patterns.  If they haven't been able to get to these then those which are non-paying will be moved to low priority after 5 days.  So this prioritization is right, we just need folks to know that there will be no newly created Low tickets.